packages/plugin/webpack/test/once_spec.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import once from '../src/util/once';
+
+describe('Once', () => {
+  it('triggers wrapped function', () => {
+    const fakeA = sinon.fake();
+    const fakeB = sinon.fake();
+    const [wrappedA] = once(fakeA, fakeB);
+    wrappedA();
+    expect(fakeA.called).to.equal(true);
+    expect(fakeB.called).to.equal(false);
+  });
+  it('triggers only once', () => {
```

Comment:
Nit (formatting): I prefer space between `it` blocks
```suggestion
  });

  it('triggers only once', () => {
```